src/main/resources/swagger.api/schemas/common.yaml

```diff
@@ -9,7 +9,6 @@ Metadata:
   properties:
     createdDate:
       type: string
-      format: date-time
       description: Date and time when the record was created
```

Comment:
in order to prevent time zone shifting, When LocalDateTime -> Date time zone shifting 3 hours